projects/solar/src/relay_fsm.c

```diff
@@ -18,8 +18,11 @@
 static SolarMpptCount s_solar_mppt_count;
 
 static void prv_assert_relay(SoftTimerId timer_id, void *context) {
-  if (((RelayFsmStorage *)context)->is_error_called) {
-    fault_handler_raise_fault(EE_SOLAR_RELAY_OPEN_ERROR, 0);
+  RelayFsmStorage *storage = context;
+
+  // Checking if a relay open error has been raised
+  if (storage->is_error_called) {
+    fault_handler_raise_fault(EE_SOLAR_RELAY_ERROR_CURRENT_EXCEEDED_NOT_OPEN, 0);
```

Comment:
Similarly here the first argument should be `EE_SOLAR_RELAY_OPEN_ERROR` and the second should be `EE_SOLAR_RELAY_ERROR_CURRENT_EXCEEDED_NOT_OPEN`.

Reply:
done